### DISCUSSION TITLE
Table styling

### DISCUSSION THREAD

**Discussion by gaynorc** [nhsevidence/NICE-Experience#185] [General]
We've been talking about tables and wondered if there's option to use something similar to the [GDS table style](https://design-system.service.gov.uk/components/table/). It's clean, accessible, user tested and open source!
This would be in addition to the existing striped table. There are still some questions around the bigger tables we have on the site and if they'd work with the GDS style one. But for things like the gender pay gap page the more minimal table would work so much better.

**Comment by wa-rren-dev** [comment]
Would need to go through the UX folks but based on the range of tables I've seen I think there would be room for a minimal table alternative when it's below a certain number of columns / rows perhaps? Any thoughts @Jbailey550?

**Reply by rnickr** [reply]
@gaynorc @wa-rren-dev - Be interested in picking this up, i've been working with Henry and Behrooz on data dashboard stuff and been looking into table design for that, defo think having a simpler/cleaner table version could be useful in some situations

@gaynorc - can you pull together some examples of tables on the site that we could look to play around with.

**Reply by gaynorc** [reply]
@rnickr - here are 3 examples. 
[Gender pay gap page](https://www.nice.org.uk/about/who-we-are/corporate-publications/gender-pay-gap-report) has a couple of small tables.
[Medtech static list](https://www.nice.org.uk/about/what-we-do/our-programmes/nice-guidance/medical-technologies-guidance/medical-technologies-guidance-static-list) is a bit bigger.
[Join a committee](https://www.nice.org.uk/get-involved/our-committees/join-a-committee) has 2 tables that vary in length depending on the number of vacancies.
If you need anymore, give me a shout and I'll dig some out.

**Comment by rnickr** [comment]
Initial draft of how this might work has been done: https://xd.adobe.com/view/45050aa7-a5f5-4e26-9882-ee1c84127e6a-1056/

Needs to be made responsive before looking to build it out.
